### PR TITLE
increase PromtailDown 'for' value

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
Towards https://github.com/giantswarm/giantswarm/issues/30274

This PR increases PromtailDown "for" value to avoid excessive and oftentimes useless paging.

### Checklist

- [x] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ spec:
       rules:
         - alert: PromtailDown
           annotations:
-            description: '{{`Scraping of all promtail pods to check if one failed every 5 minutes.`}}'
+            description: '{{`Scraping of all promtail pods to check if one failed every 30 minutes.`}}'
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml:14]: Did you check the ksm node metrics? Also, I'm a bit unsure what happens if the container keeps restarting every 2-5 minutes

Comment (QuantumEnigmaa) [reply helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml:14]: > Did you check the ksm node metrics?

No, what would it be for ?

> Also, I'm a bit unsure what happens if the container keeps restarting every 2-5 minutes

I think we already had this issue with other apps and decided to have 2 different alerts : 1 for when the app is litterally down, and another for when it keeps restarting and/or crashlooping

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml:14]: something like that I guess ?

(kube_pod_info * on (cluster_id, pod) group_left() up{container="promtail"}) * on (cluster_id, node) kube_node_status_condition{condition="Ready", status="true"}

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml:14]: But let's check with herve :D

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**